src/items/generics.md

```diff
@@ -2,30 +2,27 @@
 
 > **<sup>Syntax</sup>**\
 > _Generics_ :\
```

Comment:
Since this one is just for parameters, saying it's generics in general feels like an overreach.